app-creator/README.md

```diff
@@ -0,0 +1,81 @@
+# Earth Engine App Creator
+
```

Comment:
Add a two line description of the purpose of the EE App Creator and what it's meant to do.